README.md

```diff
@@ -45,8 +45,9 @@ group :development do
 end
 ```
 
-If using VS Code, install the [Ruby LSP extension](https://github.com/Shopify/vscode-ruby-lsp) to get the extra features in
-the editor.
+If using VS Code, install the [Ruby LSP extension](https://github.com/Shopify/vscode-ruby-lsp) to get the extra features
+in the editor. See [editors](https://github.com/Shopify/ruby-lsp/blob/main/EDITORS.md) for community instructions on
```

Comment:
```suggestion
in the editor. See [editors](EDITORS.md) for community instructions on
```

Reply:
I tried this locally and it doesn't seem to work. Does it require some more YARD configuration?

Reply:
Ah sorry yes, if you want the link to work in the Yardoc page and in the GitHub README then you'd need the full URL.